examples/recommendations/get_recommendation_impact_metrics.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example is to get impact metrics for a custom budget. 
+
+Use this example to get impact metrics for a budget amount during campaign creation workflows.
+
+This example uses the following: 
+1) Performance Max for the campaign type
+2) United States for the geo targeting 
+3) Maximize Conversions Value for the bidding strategy
+
+To get budget recommendations, run generate_budget_recommendations.py.
+"""
+
+
+import argparse
+import sys
+
+from google.ads.googleads.client import GoogleAdsClient
+from google.ads.googleads.errors import GoogleAdsException
+
+
+def main(client, customer_id, budget_amount_user):
```

Comment:
budget_amount_user is a little bit of a confusing variable name, to me. Can we just say "budget_amount"? Or if you feel like it needs to be said that it's a user value, "user_provided_budget_amount"?

Reply:
Perfect, good suggestion. I'll change it to "user_provided_budget_amount" so it's clearer.